addons/hr/hr.py

```diff
@@ -316,10 +316,11 @@ def onchange_department_id(self, cr, uid, ids, department_id, context=None):
         return {'value': value}
 
     def onchange_user(self, cr, uid, ids, user_id, context=None):
-        work_email = False
         if user_id:
-            work_email = self.pool.get('res.users').browse(cr, uid, user_id, context=context).email
-        return {'value': {'work_email': work_email}}
+            user_obj = self.pool.get('res.users').browse(cr, uid, user_id, context=context)
```

Comment:
I would also write user.image on employee.image, instead of using image_medium. Other fields are automatically computed.